### PR TITLE
Series code for Assignment 2

### PR DESCRIPTION
**Pull request by rfelts**
Here is the Fibonacci and Lucas series code problem for assignment 2. Based on the assignment description I wrote the code using recursive functions. Which becomes increasing inefficient as higher series values are requested. But as I'm guessing the assignment was designed to teach how recursion works in coding. I did not switch to another method like iteration to see if it was more efficient.

Russell

### CODE REVIEW

**Comment by natasha-aleksandrova** [comment]
@rfelts good observation on recursion efficiency here 😉

File: students/rfelts/lesson02/series.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    Returns:
+    int: The nth value in the fibonacci series
+
+    """
+
+    if n > 1:
+        return fibonacci(n - 1) + fibonacci(n - 2)
+    else:
+        return n
+
```
Comment (charlesrobison) [inline students/rfelts/lesson02/series.py]: Nice and concise!

File: students/rfelts/lesson02/series.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    int: The nth value in the lucas series
+
+    """
+
+    if n == 0:
+        return 2
+    elif n == 1:
+        return 1
+    else:
+        return lucas(n - 1) + lucas(n - 2)
```
Comment (charlesrobison) [inline students/rfelts/lesson02/series.py]: Might consider adding a catch for negative number inputs.

File: students/rfelts/lesson02/series.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+
+def sum_series(n, seriesA=0, seriesB=1):
```
Comment (natasha-aleksandrova) [inline students/rfelts/lesson02/series.py]: note that per python style guide, var names (and functions) should follow snake_case convention -> `series_a`

**Pull request merged**